stumpy/motifs.py

```diff
@@ -329,14 +335,22 @@ def motifs(
 
 @core.non_normalized(
     aamp_match,
-    exclude=["normalize", "M_T", "Σ_T", "T_subseq_isfinite", "p"],
+    exclude=[
+        "normalize",
+        "M_T",
+        "Σ_T",
+        "T_subseq_isfinite",
+        "T_subseq_isconstant",
+        "p",
+    ],
     replace={"M_T": "T_subseq_isfinite", "Σ_T": None},
 )
 def match(
     Q,
     T,
     M_T=None,
     Σ_T=None,
+    T_subseq_isconstant=None,
     max_distance=None,
     max_matches=None,
     atol=1e-8,
```

Comment:
Since `match` is  a public API, I should put `T_subseq_isconstant=None` at the end. However, this might look like a little bit weird as we then put it after `atol` param. That is why I put it right after `Σ_T`. Should I break the backward compatibility here?

Reply:
No, please put it after `atol`

Reply:
Sure. I will revise it.